pkg/client/rules.go

```diff
@@ -12,15 +12,30 @@ import (
 	"github.com/grafana/cortex-tools/pkg/rules/rwrulefmt"
 )
 
+const (
+	rulerAPIPath  = "/api/v1/rules"
+	legacyAPIPath = "/api/prom/rules"
+)
+
 // CreateRuleGroup creates a new rule group
 func (r *CortexClient) CreateRuleGroup(ctx context.Context, namespace string, rg rwrulefmt.RuleGroup) error {
 	payload, err := yaml.Marshal(&rg)
 	if err != nil {
 		return err
 	}
 
+	path := rulerAPIPath
+	if r.legacy {
+		path = legacyAPIPath
+	}
 	escapedNamespace := url.PathEscape(namespace)
-	res, err := r.doRequest("/api/prom/rules/"+escapedNamespace, "POST", payload)
+	path += "/" + escapedNamespace
+
+	log.WithFields(log.Fields{
+		"url": path,
+	}).Debugln("path built to request rule group")
```

Comment:
**nitpick:** I think the `doRequest` function logs the path of the request. If so, this log line should be removed.

Reply:
I agree. I'll remove it.